src/beacon/app/main.py

```
@@ -1,17 +1,13 @@
-from fastapi import FastAPI, Depends, HTTPException, status, BackgroundTasks, Query
+from fastapi import FastAPI, Depends, HTTPException, Query
 from fastapi.middleware.cors import CORSMiddleware
-from fastapi.responses import JSONResponse
-from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from fastapi.responses import JSONResponse, Response
```

Ruff (0.12.2)

`fastapi.responses.JSONResponse` imported but unused

Remove unused import: `fastapi.responses.JSONResponse`

(F401)